.github/workflows/check_issue_label.yml

```diff
@@ -0,0 +1,115 @@
+
+name: Check Issue Labels
+
+on:
+  issues:
+    types:
+      - labeled
+
+env:
+  PAT_TOKEN: ${{ secrets.TEAM_READ_PAT }}  # PAT with read:org scope
+
+permissions: {}
+
+jobs:
+  check-label:
+    runs-on: ubuntu-22.04
+    permissions:
+      issues: write
+
+    steps:
+      - name: Validate issue label
+        uses: actions/github-script@v7
+        with:
+          github-token: ${{ env.PAT_TOKEN }}
+          script: |
+            const { context } = require('@actions/github');
+            const octokit = require('@octokit/rest').Octokit.defaults({
+              auth: process.env.PAT_TOKEN
+            });
+            // ======================
+            // SECURE INPUT HANDLING
+            // ======================
+            const sanitizedLabel = (context.payload.label?.name || '').trim();
+            const actorUsername = (context.actor || '').trim().toLowerCase();
+            const issueNumber = Number(context.issue.number) || 0;
+            const { owner, repo } = context.repo;
+            // Validation checks
+            if (!/^[\w\s-:]+$/i.test(sanitizedLabel)) {
+              throw new Error(`Invalid label format: ${sanitizedLabel}`);
+            }
+            if (!/^[a-z\d-]+$/.test(actorUsername)) {
+              throw new Error(`Invalid username: ${actorUsername}`);
+            }
+            if (!Number.isSafeInteger(issueNumber) || issueNumber < 1) {
+              throw new Error(`Invalid issue number: ${issueNumber}`);
+            }
+            // ======================
+            // TEAM MEMBERSHIP CHECK
+            // ======================
+            async function getTeamMembers() {
+              try {
+                const result = await octokit.graphql({
+                  query: `query ($org: String!, $teamSlug: String!) {
+                    organization(login: $org) {
+                      team(slug: $teamSlug) {
+                        members(first: 100) {
```

Comment:
we should throw an error if the team has more than 100 members. This probably won't ever happen, but it's always good to validate assumptions like this and throw noisy errors if they're violated